### ISSUE TITLE
Loss curve

### ISSUE THREAD

**Issue by Mohammad-GHK** [matterport/Mask_RCNN#2919]
Hello everyone.
can anyone tell me how can I plot loss curve in training phase.
I'm using model.train

thanks.

**Comment by Kolmobatem** [comment]
Hi ! I am having the same issue ! The loss is displayed but how can you retrieve the loss curves ?

**Comment by elliestath** [comment]
Hi, if you still have this problem, here's my idea.
You need to add a `history` parameter after your `model.train()`

```
model.train(dataset_train, dataset_val, 
            #learning_rate=config.LEARNING_RATE,
            epochs=30, 
            layers='heads',
            augmentation=augmentation)

history = model.keras_model.history.history
```

After that, you can plot them like:

```
# Plot training & validation loss values
plt.figure(figsize=(15, 5))
plt.plot(history['loss'])
plt.plot(history['val_loss'])
plt.title('Model loss')
plt.ylabel('Loss')
plt.xlabel('Epoch')
plt.legend(['Train', 'Validation'], loc='upper left')
plt.savefig('loss.png', dpi=300, bbox_inches='tight')
```
